src/main/java/org/terasology/launcher/updater/LauncherUpdater.java

```diff
@@ -60,13 +59,15 @@ private Semver versionOf(GitHubRelease release) {
      * <br>
      * Compares the current launcher version number to the upstream version number if an internet connection is available.
      *
-     * @return a {@link GitHubRelease} if an update is available, null otherwise
+     * @return a {@link GHRelease} if an update is available, null otherwise
      */
     //TODO: return Option<GitHubRelease>
-    public GitHubRelease updateAvailable() {
+    public GHRelease updateAvailable() {
         //TODO: only check of both version are defined and valid semver?
         try {
-            final GitHubRelease latestRelease = github.getLatestRelease("movingblocks", "terasologylauncher");
+            final GitHub github = GitHub.connectAnonymously();
+            final GHRepository repository = github.getRepository("MovingBlocks/TerasologyLauncher");
+            final GHRelease latestRelease = repository.getLatestRelease();
```

Comment:
This was all it took to change API libraries? I expected this to be more work!

But I guess if the only API function we're using is "get latest release", there's not much to it 😃

Reply:
Well, I purposefully designed our little Github model to be close to this other library. That was actually my first choice, but due to a bug did not work. Interaction went quite well, bug is fixed, and we're all set! 👍 

I _think_ this is doing two request instead of one, though. Should not be a huge deal, but something we should keep in mind...